From 3822bda0cb6510a7fbb3728d23ad4192b09967b8 Mon Sep 17 00:00:00 2001
From: "Vladimir S. FONOV" <vladimir.fonov@gmail.com>
Date: Wed, 15 Jul 2015 15:50:07 -0400
Subject: [PATCH] COMP: Fixes issue when MINCTransformation module is not
 initialized

This issue affects only older gcc compilers when compiling 64 bit
affects gcc 4.4 on Debian 6, CentOS 6 , Ubuntu 10.04

The MINCIOTransform module does register , but then does
not get included in the list of available transformation IO modules.
Maybe it could be fixed by splitting up MINC module into
MINC and MINCTransform

Change-Id: I4e887d7f61746cb7573877c2c27a122f354dce15
---
 Modules/IO/MINC/itk-module.cmake | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Modules/IO/MINC/itk-module.cmake b/Modules/IO/MINC/itk-module.cmake
index cf331b0..cae7267 100644
--- a/Modules/IO/MINC/itk-module.cmake
+++ b/Modules/IO/MINC/itk-module.cmake
@@ -2,7 +2,6 @@ set(DOCUMENTATION "This module contains classes for reading and writing image
 files in the MINC 2 file Format (mnc) and transformations (xfm).")
 
 itk_module(ITKIOMINC
-  ENABLE_SHARED
   DEPENDS
     ITKMINC
     ITKIOImageBase
-- 
1.9.1

